tests/integration/explain/default/delete_test.go

```diff
@@ -292,7 +292,12 @@ func TestDefaultExplainMutationRequestWithDeleteUsingNoIds(t *testing.T) {
 							"collectionID":   "3",
 							"collectionName": "Author",
 							"filter":         nil,
-							"spans":          []dataMap{},
+							"spans": []dataMap{
+								{
+									"end":   "/4",
+									"start": "/3",
+								},
+							},
```

Comment:
question: So this was the bug that was caused due to the way `HasValue` was being used?

Reply:
Yeah, there was something fishy going on here with HasValue.  The newly expected result is correct, and the old property/type has died so I didn't bother chasing down exactly where the bug was